### PR TITLE
Edit landing page

### PR DESCRIPTION
**Pull request by leikkisa**
connects #7268

### CODE REVIEW

File: app/views/higher_level_reviews/edit.html.erb [hunk 1]
```diff
@@ -4,9 +4,6 @@
     dropdownUrls: dropdown_urls,
     feedbackUrl: feedback_url,
     buildDate: build_date,
-    formType: "higher_level_review",
-    review: higher_level_review.ui_hash,
-    ratings: higher_level_review.cached_serialized_timely_ratings,
-    ratedRequestIssues: higher_level_review.request_issues.rated.map(&:ui_hash)
+    serverIntake: higher_level_review.ui_hash(FeatureToggle.enabled?(:intakeAma, user: current_user))
```
Comment (jiahuang) [inline app/views/higher_level_reviews/edit.html.erb]: 👍 looks cleaner this way

File: client/app/intake/IntakeFrame.jsx [hunk 2]
```diff
@@ -90,7 +89,7 @@ class IntakeFrame extends React.PureComponent {
                   exact
                   path={PAGE_PATHS.ADD_ISSUES}
                   title="Add Issues | Caseflow Intake"
-                  component={AddIssues} />
+                  component={IntakeAddIssuesPage} />
```
Comment (jiahuang) [inline client/app/intake/IntakeFrame.jsx]: 👍 on the naming consistency

**Comment by leikkisa** [comment]
edit-modals

**Comment by leikkisa** [comment]
> Also note for the future -- we're testing the same functionality on the issues page a lot (3x for regular intake on hlr, appeal & sc, and now another 2x for editing hlr and sc). Eventually, we should break this out into js tests or something so it only gets repeated once. The ruby feature specs would then just test a simple path of adding one issue.

I think that's a good call! It was really useful for me in this case, testing helped me find things that were broken in intake vs. edit and HLR vs SC.  Happy to start converting to JS tests in the future.

**Pull request merged**